evennia/utils/utils.py

```diff
@@ -472,6 +472,31 @@ def iter_to_str(iterable, sep=",", endsep=", and", addquote=False):
 list_to_string = iter_to_str
 iter_to_string = iter_to_str
 
+re_empty = re.compile("\n\s*\n")
+
+def strip_extra_whitespace(text, max_linebreaks=1, max_spacing=2):
```

Comment:
I think the naming of this function is confusing, the `strip` name does not make it clear this is also removing whitespace _inside_ a string. Maybe a name like `limit_whitespace_in_string` would be clearer?